from Exceptions.baseException import BaseException


class UserNotFoundException(BaseException):
    def __init__(self):
        super().__init__()
        self.message = "No se encontr√≥ un usuario"
        self.statusCode = 404
